app/javascript/vue/navigation/notifications/notifications_flyout.vue

```diff
@@ -39,20 +39,25 @@ export default {
       notifications: [],
       nextPage: 1,
       scrollBar: null,
-      loadingPage: false
+      loadingPage: false,
```

Comment:
trailing comma

Reply:
I think this changed with the new linter config, maybe we should set it to not allow trailing commas? Some people like to have them, some don't.